vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/VGSCheckoutAddCardConfig.kt

```diff
@@ -52,30 +53,32 @@ class VGSCheckoutMultiplexingAddCardConfig private constructor(
     /**
      * Public constructor.
      *
-     * @param accessToken multiplexing app access token.
-     * @param tenantId payment orchestration tenant id.
+     * @param accessToken payment orchestration app access token.
+     * @param tenantId unique organization vault id.
      * @param environment type of vault.
      * @param formConfig UI configuration.
      * @param isScreenshotsAllowed If true, checkout form will allow to make screenshots. Default is false.
      * @param isAnalyticsEnabled If true, checkout will send analytics events that helps to debug
      * issues if any occurs. Default value is true.
      *
-     * @throws com.verygoodsecurity.vgscheckout.exception.VGSCheckoutException if access token is not valid.
+     * @throws com.verygoodsecurity.vgscheckout.exception.VGSCheckoutJWTParseException if access token is not valid.
+     * @throws com.verygoodsecurity.vgscheckout.exception.VGSCheckoutJWTRestrictedRoleException if
+     * access token is contains restricted roles.
      */
     @JvmOverloads
-    @Throws(VGSCheckoutException::class)
```

Comment:
@Endorf Should be `VGSCheckoutException` like in `VGSCheckoutPaymentConfig`? 🤔

Reply:
Thanks! It might be an artifact after merging.